src/Microsoft.AspNetCore.Authentication/VirtualSchemeOptions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Used to redirect authentication methods to another scheme
+    /// </summary>
+    public class VirtualSchemeOptions
+    {
+        public string DefaultTarget { get; set; }
```

Comment:
These names bug me

Reply:
Yeah not in love with the current Target naming either, but DefaultScheme didn't seem to great...    Redirect/Forward were other names I considered...

Reply:
So names...  could just go back to Schemes I guess with VirtualSchemeOptions having Authenticate/Challenge/Forbid/SignIn/SignOutScheme properties.